oauth_provider/migrations/0001_initial.py

```diff
@@ -66,9 +67,9 @@ class Migration(migrations.Migration):
                 ('verifier', models.CharField(max_length=10)),
                 ('callback', models.CharField(max_length=2083, null=True, blank=True)),
                 ('callback_confirmed', models.BooleanField(default=False)),
-                ('consumer', models.ForeignKey(to='oauth_provider.Consumer')),
-                ('scope', models.ForeignKey(blank=True, to='oauth_provider.Scope', null=True)),
-                ('user', models.ForeignKey(related_name=b'tokens', blank=True, to=settings.AUTH_USER_MODEL, null=True)),
+                ('consumer', models.ForeignKey(to='oauth_provider.Consumer',on_delete=models.DO_NOTHING)),
```

Comment:
while testing, on newer versions of `django` the `on_delete` attribute is required, [it was causing runtime errors]

Reply:
@feanil any concerns on this ?